cves/kernel/CVE-2012-6545.yml

```diff
@@ -181,8 +178,11 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  note: |
+    This was a simple memory leak, so some kind of memory leak discoverer could 
```

Comment:
Consider changing "some kind of memory leak discoverer" to "memory profiler".

Reply:
Can do.